packages/@aws-cdk/aws-lambda/lib/lambda-ref.ts

```diff
@@ -181,6 +183,23 @@ export abstract class FunctionRef extends cdk.Construct
         });
     }
 
+    /**
+     * Convenience method for creating a new {@link PipelineInvokeAction},
+     * and adding it to the given Stage.
+     *
+     * @param stage the Pipeline Stage to add the new Action to
+     * @param name the name of the newly created Action
+     * @param props the properties of the new Action
+     * @returns the newly created {@link PipelineInvokeAction}
+     */
+    public addToPipeline(stage: codepipeline.IStage, name: string, props?: CommonPipelineInvokeActionProps): PipelineInvokeAction {
+        return new PipelineInvokeAction(this.parent!, name, {
```

Comment:
Didn't we agree that we always use `this` here?

Reply:
True. Will change.